server/core/src/main/java/io/whitefox/core/services/FileIOFactory.java

```diff
@@ -0,0 +1,9 @@
+package io.whitefox.core.services;
+
+import io.whitefox.core.Metastore;
+import io.whitefox.core.Storage;
+import org.apache.iceberg.io.FileIO;
+
+public interface FileIOFactory {
```

Comment:
Are you sure Iceberg does not offer this functionality out of the box?

Reply:
I checked now but I cannot find a factory-like class for FileIO.

Reply:
I found it and added it :smile: